plugins/acpi-wpbt/README.md

```diff
@@ -0,0 +1,15 @@
+ACPI WPBT
+=========
+
+Introduction
+------------
+
+This plugin checks if the WPBT table has been exported. Although unused by
+Linux, Microsoft Windows will unconditionally execute the EXE file specified by
+the WPBT table. WPBT should be disabled in the firmware setup menu, and it
+certainly should not be enabled by default at the factory.
```

Comment:
Does any manufacturer call it this in BIOS setup?

Reply:
IIRC, it's exported when you enable Computrace on Lenovo.

Reply:
Right, my point was that I don't believe any manufacturer exports this to a user with the phrase `WPBT`, that's just the means to the end to enable a technology.

Most manufacturers export it as `Computrace` and that is probably what should be referenced here but just should be worded carefully not to step on the hypothetical toes by calling out their software as bad.  
It's just the way that the software and infrastructure works and some admins may choose to have it as an element of their security threat model.

Reply:
I've reworded things a bit, now about that?